apps/op-app/src/use-cases/send-event-messge.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  Session,
+  SessionEnvironment,
+  eventsSchema,
+  getEvent,
+  getSession,
+  writeEvent,
+} from "@/domain/session.js";
+import { StorageEnvironment, sendEventsMessage } from "@/domain/storage.js";
+import * as E from "fp-ts/lib/Either.js";
+import * as O from "fp-ts/lib/Option.js";
+import * as RTE from "fp-ts/lib/ReaderTaskEither.js";
+import { flow, pipe } from "fp-ts/lib/function.js";
+import {
+  auditEventSchema,
+  requestParamsSchema,
+} from "io-fims-common/domain/audit-event";
+import * as jose from "jose";
+import * as assert from "node:assert/strict";
+import { z } from "zod";
+
+const eventParamsSchema = z.discriminatedUnion("type", [
```

Comment:
I think `event` it is not specific enough. You should replace it with something like `auditEvent` (and its variations `AUDIT_EVENT`, `AuditEvent`)